_maps_2024/28-aki.md

```diff
@@ -0,0 +1,15 @@
+---
+day: 28
+creator: Akira Di Sandro
+title: The Great Lakes
+thumbnail: /assets/thumbnails/2024/28-BluePlanet.png
+source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day28.R
+---
+
+I grew up in the Midwest, so I wanted to showcase the Great Lakes for today's map challenge. With it also being Thanksgiving (a holiday that whitewashes this country's genocidal history), I wanted to acknowledge the people that lived on the land before colonization.
+
+Finding and mapping data regarding where Indigenous Americans lived proved to be much more challenging than expected. I pulled information from multiple sources and estimated locations of different tribes to the best of my ability based on available information.
+
+Check out this [cool website](https://native-land.ca/) that visualizes indigenous peoples around the world.
+
+![Screenshot of map](assets/thumbnails/2024/28-BluePlanet.png)
```

Comment:
```suggestion
![Screenshot of map](assets/thumbnails/2024/28-Aki-BluePlanet.png)
```

Reply:
thank you for catching that! clearly finishing a map at 3 am took its toll.....😅